src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,23 @@
+.stars {
+  display: flex;
+
+  &__star {
+    background-image: url("../images/star.svg");
+    background-position: center;
+    background-repeat: no-repeat;
+    margin-right: 4px;
+    width: 16px;
+    height: 16px;
+    line-height: 140%;
+  }
+
+  @for $i from 1 through 5 {
+    &--#{$i} &__star:nth-child(-n + #{$i}) {
+      background-image: url("../images/star-active.svg");
+    }
+  }
+
+  &__star:last-child {
+    margin: 0;
+  }
```

Comment:
1) nest styles for `:last child` 
2) typically we state all related styles and then use loops and similar scss stuff.